docs/files/templates/lx-instance/watchmaker-lx-instance.template

```diff
@@ -852,7 +858,14 @@
                                     { "Ref" : "AWS::Region" }, " &&",
                                     " chown root:root /etc/cfn/scripts/make-app &&",
                                     " chmod 700 /etc/cfn/scripts/make-app"
-                                ]]}
+                                ]]},
+                                "env" :
+                                {
+                                  "AWS_DEFAULT_REGION" : { "Ref" : "AWS::Region" },
```

Comment:
Since we're passing in the region via the env, we could remove it from the aws s3 cp command itself.

Reply:
And this block is missing in the autoscale edits...

Reply:
fixed